### ISSUE TITLE
Implement Photos App

### ISSUE THREAD

**Issue by provokateurin** [nextcloud/neon#355]

**Comment by provokateurin** [comment]
@vauvenal5 it looks like you are not working on Yaga anymore, right?

@Leptopoda we could try to port https://github.com/vauvenal5/yaga to neon. I don't know the codebase and how much would need to be done, but I think it's a very good starting point.

**Comment by Leptopoda** [comment]
I'll have a look after #353

**Comment by vauvenal5** [comment]
I haven't abandoned the project. Just was very busy lately.

**Comment by provokateurin** [comment]
Ah alright :) Would you mind taking a look at this project and see if it makes sense for you to adopt it/become a part of it? The whole project is still moving very fast, but I think we are getting into a really good spot now where people can actually start using it.

**Comment by provokateurin** [comment]
It would also be great to support the API of the [memories app](https://github.com/pulsejet/memories), it seems to be much faster than the photos app (which uses the slow webdav).

**Comment by provokateurin** [comment]
For implementing this it doesn't really matter if the photos app is installed on the server since the app only needs WebDAV which is always available. We don't support apps that do not directly correspond an app on the server yet though.

**Comment by vauvenal5** [comment]
This is actually very good point also in context of something I will be doing next. I want to provide album support in Yaga. However, as far as I saw it albums are not actually provided by the webdav api but specifically by the photos api. So therefor a custom endpoint would be needed anyway in the Nextcloud client.

**Comment by provokateurin** [comment]
I see, but it would still be nice to support "photos" even when the photos app is not installed on the server.

**Comment by vauvenal5** [comment]
Yes, everything else is supported by the basic webdav api as already provided in the Nextcloud package. My entire communication is going over your package.

**Comment by vauvenal5** [comment]
I am currently working on this, not sure though, how to assign it to me.
